Sources/CasePaths/EnumReflection.swift

```diff
@@ -11,6 +11,13 @@ extension CasePath {
       extract: CasePaths.extract(embed)
     )
   }
+
+  public static func optionalCase<Wrapped>(_ embed: @escaping (Value) -> Wrapped?) -> CasePath where Root == Wrapped? {
```

Comment:
Can this be an overload, as well?
```suggestion
  public static func `case`<Wrapped>(_ embed: @escaping (Value) -> Wrapped?) -> CasePath where Root == Wrapped? {
```

Reply:
I tried that and it didn't work.

Reply:
Could be we need to specify generics differently. If you can remember a case that didn't work I can mess with it.

Reply:
Rename it to `case` and change the `/` overload that calls it. In Xcode 13 beta 3, the `/` operator then calls the wrong `case` overload.